grails-app/controllers/org/broadinstitute/orsp/api/NewConsentGroupController.groovy

```diff
@@ -39,6 +38,10 @@ class NewConsentGroupController extends AuthenticatedController {
         render(view: "/mainContainer/index", model: [projectKey: params.projectKey, consentKey: params.consentKey, issueType: 'consent-group'])
     }
 
+    def redirect() {
```

Comment:
nit: confusing name. This doesn't actually redirect, it just renders a view.

Reply:
I've changed this to `renderMainComponent`.